libs/blocks/modal/modal.js

```diff
@@ -49,6 +49,18 @@ export function sendAnalytics(event) {
   }
 }
 
+function focusAfterModalClose(modal) {
+  const { id } = modal;
+  const isGeoPopup = id === 'locale-modal-v2';
```

Comment:
Not sure we need `id` here, it's only used once. Code can be rewritten as `modal?.id === LOCALE_MODAL_ID`